src/Workleap.OpenApi.MSBuild/CodeFirstProcess.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Workleap.OpenApi.MSBuild;
+
+/// <summary>
+/// For a Code First approach it will:
+///     1. (if not disabled) Generate the OpenAPI specification files from the code
+///     2. Validate the OpenAPI specification files base on spectral rules 
+/// </summary>
+internal class CodeFirstProcess
+{
+    private readonly SpectralManager _spectralManager;
+    private readonly SwaggerManager _swaggerManager;
+
+    internal CodeFirstProcess(SpectralManager spectralManager, SwaggerManager swaggerManager)
```

Comment:
This file looks good overall. Minor comments:

`Environment.GetEnvironmentVariable("WL_DISABLE_SPECGEN") != "true";`
Usually, strings from env vars are compared using ordinal ignore case.

`isGenerationEnable` => `isGenerationEnabled` (+d).

If manually copy pasting the generated spec file from bin/ to the source code dir, handle `IOException` than can occur if the file is already opened and locked by the dev IDE. Print a better error message, followed by the original exception